<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Inicio</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lbImg">
    <property name="geometry">
     <rect>
      <x>-4</x>
      <y>-8</y>
      <width>650</width>
      <height>650</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>650</width>
      <height>650</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>650</width>
      <height>650</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../img/inicio.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btIniciar">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>550</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Ink Free&quot;;
background-color: white;
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>Iniciar</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
